From 16df5432a6b073f777cacde4753919c13135e4ad Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Tue, 8 Mar 2022 15:37:09 +0530
Subject: [PATCH] Update dependecy path for bt

sytem/bt is moved to packages/modules/Bluetooth

system/bt/device/include -> packages/modules/Bluetooth/system/device/include
system/bt/stack/include -> packages/modules/Bluetooth/system/stack/include/

Error:
module android.hardware.bluetooth@1.0-service.vbt missing
dependencies: system/bt/device/include, system/bt/stack/include

Signed-off-by: svenate <salini.venate@intel.com>
---
 bluetooth/Android.bp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bluetooth/Android.bp b/bluetooth/Android.bp
index c6611a9..343301f 100644
--- a/bluetooth/Android.bp
+++ b/bluetooth/Android.bp
@@ -18,8 +18,8 @@ cc_binary {
     proprietary: true,
     relative_install_path: "hw",
     include_dirs: [
-        "system/bt/device/include",
-        "system/bt/stack/include"
+	"packages/modules/Bluetooth/system/device/include",
+	"packages/modules/Bluetooth/system/stack/include"
     ],
     srcs: [
         "async_fd_watcher.cc",
-- 
2.17.1

